src/components/form/form.stories.mdx

```diff
@@ -1,25 +1,8 @@
 import { Meta, Story, Canvas } from "@storybook/addon-docs";
-import { useState } from "react";
 import TranslationKeysTable from "../../../.storybook/utils/translation-keys-table";
 import StyledSystemProps from "../../../.storybook/utils/styled-system-props";
-
 import Form from ".";
-import Textbox from "../textbox";
-import Textarea from "../textarea";
-import { Tab, Tabs } from "../tabs";
-import { RadioButton, RadioButtonGroup } from "../radio-button";
-import DateInput from "../date";
-import { Select, MultiSelect, Option } from "../select";
-import { Checkbox } from "../checkbox";
-import InlineInputs from "../inline-inputs";
-import Hr from "../../components/hr";
-import Switch from "../switch";
-import Button from "../button";
-import DialogFullScreen from "../dialog-full-screen";
-import Dialog from "../dialog";
-import Box from "../box";
-import CarbonProvider from "../carbon-provider/carbon-provider.component";
-
+import * as stories from "./form.stories";
 
 <Meta title="Form" parameters={{ info: { disable: true } }} />
```

Comment:
```suggestion
<Meta title="Form" parameters={{ info: { disable: true }, controls: { disable: true } }} />
```
We will want to disable storybook controls for these stories since QAs will not be needing them here.